### PR TITLE
homepage added

### PR DESCRIPTION
**Pull request by swstica**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+                <div class="nav-item">Beauty</div>
+                <div class="search-container">
+                    <form>
+                      <button type="submit"><i class="fa fa-search"></i></button>
+                      <input type="text" placeholder="Search.." name="search">
+                    </form>
+                  </div>
+                <div class="nav-item">Profile</div>
+                <div class="nav-item">Wishlist</div>
+                <div class="nav-item">Cart</div>
```
Comment (JuvitaSaini09) [inline index.html]: just a asuggestion,you could have used anchor tag here,these are links

File: index.html [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+            <div class="btn-container"><button class="btn">Shop now</button></div>
+            <img class="img-container" src="images/main.jpg">
+        </div>
+
+        <div class="card-container">
+            <div class="flex-flow-row card-items">
+                <div class="card card-component">
+                    <img class="card-img" src="images/women.jpg">
+                    <div class="card-details flex-flow-column">
+                        <h2><a>Apparel</a></h2>
```
Comment (sri-rishi) [inline index.html:45]: in a tag provide attribute href if it's has no link give it like href="#"

File: index.css [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+    border-bottom-left-radius: 0.5rem;
+   
+  }
+  .search-container{
+      border: 1px solid var(--dark-grey-color);
+      border-radius: 0.5rem;
+  }
+
+
+/*main section*/
```
Comment (sri-rishi) [inline index.css:64]: remove extra spacing

**Review by sri-rishi — CHANGES_REQUESTED** [review]

**Review by Anurag021 — APPROVED** [review]
code looks clean and good, just one thing some of the images for cards are quite heavy (i.e. around 3.5mb), the site would be faster if they would be reduced, I make use of paint to reduce size/pixels.

**Review by prankurpandeyy — APPROVED** [review]

**Comment by prankurpandeyy** [comment]
last two card shold have some marging and padding

**Review by JuvitaSaini09 — APPROVED** [review]

**Pull request merged**